tests/catalog_responses/init/go_workspace.json

```diff
@@ -0,0 +1,782 @@
+[
```

Comment:
question nonblocking: looks like this is identical to `go_mod.json` - what's the reason for the duplication?

Reply:
The reason for the duplication is that I didn't look at the contents of the specific Go files used to do the init, I just recorded the responses for the different `init` tests

Reply:
Fixed